frontend/src/metabase/dashboard/components/Dashboard/stickyParameters.unit.spec.js

```diff
@@ -79,4 +87,49 @@ describe("updateParametersWidgetStickiness", () => {
 
     expect(setState).not.toHaveBeenCalled();
   });
+
+  describe("on small device", () => {
+    beforeEach(() => {
+      simulateSmallScreen();
+    });
+    it("makes filters not sticky if too many parameters", () => {
```

Comment:
Excuse my OCD :speak_no_evil: 

```suggestion
    });

    it("makes filters not sticky if too many parameters", () => {
```

Reply:
no-no it's great, I also noticed this, but it was too late to fix it yesterday. I even had in mind eslint rule to force such spacing